lib/create-space-api.ts

```diff
@@ -1130,6 +1130,46 @@ export default function createSpaceApi(makeRequest: MakeRequest) {
         params: { spaceId: raw.sys.id, query },
       }).then((response) => wrapScheduledActionCollection(makeRequest, response))
     },
+    /**
+     * Get a Scheduled Action in the current space by environment and ID
+     * @returns Promise with the Scheduled Action
```

Comment:
might also say what it will do in case an action doesn't exist or they don't have permission

Reply:
done 👍🏽